lib/core/src/display/rendering/transform.rs

```diff
@@ -52,16 +52,31 @@ impl Transform {
         self.translation = Vector3::new(x, y, z);
     }
 
-    /// Set Transform's scale.
+    /// Gets Transform's translation.
```

Comment:
Maybe it is not part of this review, but... there is nalgebra::Projective2/3 structure, which I think does exactly the same thing as our Transform, doesn't it?

Reply:
Our Transform struct is used to keep track of Position, Rotation and Scale separately. The CPU operates faster over the Transform this way. Once we send it to the GPU, we can convert it to homogeneous matrix.